pubsub/api/Pubsub.Samples.Tests/CreateTopicWithKinesisIngestionTest.cs

```diff
@@ -31,17 +32,8 @@ public CreateTopicWithKinesisIngestionTest(PubsubFixture pubsubFixture)
     public void CreateTopicWithKinesisIngestion()
     {
         string topicId = _pubsubFixture.RandomTopicId();
-        var (streamArn, consumerArn, awsRoleArn, gcpServiceAccount) = _pubsubFixture.RandomKinesisIngestionParams();
-        Topic createdTopic = _createTopicWithKinesisIngestionSample.CreateTopicWithKinesisIngestion(_pubsubFixture.ProjectId, topicId, streamArn, consumerArn, awsRoleArn, gcpServiceAccount);
-
-        // Confirm that the created topic and topic retrieved by ID are equal
-        Topic retrievedTopic = _pubsubFixture.GetTopic(topicId);
-        Assert.Equal(createdTopic, retrievedTopic);
-
-        // Confirm that all AWSKinesis Ingestion params are equal to expected values
-        Assert.Equal(streamArn, createdTopic.IngestionDataSourceSettings.AwsKinesis.StreamArn);
-        Assert.Equal(consumerArn, createdTopic.IngestionDataSourceSettings.AwsKinesis.ConsumerArn);
-        Assert.Equal(awsRoleArn, createdTopic.IngestionDataSourceSettings.AwsKinesis.AwsRoleArn);
-        Assert.Equal(gcpServiceAccount, createdTopic.IngestionDataSourceSettings.AwsKinesis.GcpServiceAccount);
+        var (streamArn, consumerArn, awsRoleArn, gcpServiceAccount) = _pubsubFixture.KinesisIngestionParams();
+        var exception = Assert.Throws<RpcException>(() => _createTopicWithKinesisIngestionSample.CreateTopicWithKinesisIngestion(_pubsubFixture.ProjectId, topicId, streamArn, consumerArn, awsRoleArn, gcpServiceAccount));
+        Assert.Equal(StatusCode.PermissionDenied, exception.Status.StatusCode);
```

Comment:
Same question as before regarding permissions.

If we can give the account some permissions so that we get a more explicit error that's great. Else maybe we can assert that the permissions errors are specific to the external service access/setup etc.

Do you know what the specific error messages are? (Temporarily rethrow the exception on these tests so we know exactly what happens on CI which will be different from local).